setup.cfg

```
@@ -27,6 +27,7 @@ install_requires =
     glue-core>=1.6.0
     glue-jupyter>=0.14.2
     echo>=0.5.0
+    ipykernel!=6.19.3
```

What Maarten suggested.

```suggestion
    ipykernel<6.18
```